include/seqan/bam_io/bam_alignment_record.h

```diff
@@ -607,6 +607,37 @@ getAlignmentLengthInRef(BamAlignmentRecord const & record)
     return l;
 }
 
+// ----------------------------------------------------------------------------
+// Function appendRawPod()
+// ----------------------------------------------------------------------------
+
+#if SEQAN_BIG_ENDIAN
+inline void
+ensure_little_endian(BamAlignmentRecordCore & r)
```

Comment:
Really? Please don't start mixing CamelCase and Snake case in SeqAn 2.x

Reply:
I also suggest calling it `makeLittleEndian` to be more STLish.

Reply:
> Really? Please don't start mixing CamelCase and Snake case in SeqAn 2.x

Wow, I totally didn't see this.